net/vnstat2/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vnstat2
+PKG_VERSION:=2.2
+PKG_RELEASE:=1
+PKG_MAINTAINER:=
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=vnstat-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://humdi.net/vnstat/
+PKG_HASH:=c60a7bc35e0247b0d2e628d8fc39427f2482e844c2c7b9cdbfc814463310e02c
+PKG_BUILD_DIR:=$(BUILD_DIR)/vnstat-$(PKG_VERSION)
+
+PKG_INSTALL:=1
```

Comment:
Maybe PKG_BUILD_PARALLEL could be used.

Reply:
I'll see if I can test it somehow, it's a really small package so I don't think there's much to gain however...